docs/editors-guide/strategic-refinement.md

```diff
@@ -2,17 +2,17 @@
 
 ## Summary
 
-The goal of this project is to to perform strategic refinement in Mondo and clean up the hierarchy. The first steps at this refinment is to remove the Orphanet (ORDO) grouping classes since these cause unwanted classifications in Mondo.
+The goal of this project is to perform strategic refinement in Mondo and refine the hierarchy. The first steps at this refinment is to remove the Orphanet (ORDO) grouping classes since these cause unwanted classifications in Mondo.
 
 **Justification**: Orphanet takes a different approach to classification compared to Mondo, where many disease entities are grouped together based on phenotypic features. This is causing some incorrect inferences in Mondo. 
 
 **Approach**: The approach is to obsolete many of the Orphanet grouping classes and review the resulting hierarchy to ensure proper classification of the children. This will be an incremental and iterative approach. An overview of the approach is:
```

Comment:
Typo fixed here: 0d254488

Reply:
Which section is this proposed for?

Reply:
This is for the "approach" section. IF we want to keep this section separate (see previous comment). 
The approach section would therefore look like: 

Orphanet classifies diseases based on phenotypic features, and these orphanet grouping classes are in Mondo. In the first step of the Mondo classification refinement, we targeted these "orphanet grouping classes", obsoleted them and reviewed the resulting hierarchy to ensure proper classification of the children. This will be an incremental and iterative approach. An overview of the approach is

Reply:
Let's pause on this for another ticket.